src/components/dragBoxLayer.ts

```diff
@@ -416,6 +416,12 @@ export module Components {
       this._setMovableClass();
       return this;
     }
+
+    public destroy() {
```

Comment:
Can we quickly check that this bug does not affect the `guideLineLayer` as well?